test_all.sh

```diff
@@ -66,3 +66,11 @@ if [[ $USE_OPENMP -eq 1 ]]; then
         ./openmp/main -steps 9 -type $t -kernel memory_bound -scratch 64
     done
 fi
+
+if [[ $USE_OMPSS -eq 1 ]]; then
+    export LD_LIBRARY_PATH=/usr/local/clang/lib:$LD_LIBRARY_PATH
```

Comment:
I assume this is only required when the compiler is clang? Maybe we can check that instead of doing it unconditionally?

Reply:
I think this is not required for ompss. I used it to pointing out the openmp lib, but ompss uses its own libs, so I think it can be removed.